webapp/components/BasicButton.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <a
+      :href="href"
+      class="inline-flex m-1.5 px-2.5 py-2 rounded-lg shadow-md text-white text-center
+      cursor-pointer justify-center items-center"
+      :class="(!disabled ? `bg-${color}-500 hover:bg-${color}-600` : `bg-gray-500`) + (bold ? ' font-semibold' : '')">
```

Comment:
PurgeCSS will not be able to pick up the class names here. Don't use string interpolation like that in tailwind, instead, enumerate the classes.